sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+        domain="[('state', 'not in', ['done', 'cancel'])]",
+    )
+
+    partner_id = fields.Many2one(
+        "res.partner",
+        string="Customer",
+        store=True,
+        compute="_compute_partner_id",
+    )
+
+    @api.depends("sale_order_id")
+    def _compute_partner_id(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_compute_partner_id`

Add return type annotation: `None`

(ANN202)